src/oemof/solph/_models.py

```diff
@@ -446,29 +506,38 @@ def _add_parent_block_variables(self):
         for o, i in self.FLOWS:
             if self.flows[o, i].nominal_value is not None:
                 if self.flows[o, i].fix[self.TIMESTEPS.at(1)] is not None:
+                    # if nominal_value and fix is given,
+                    # set flow to fix * nominal value
```

Comment:
I think removing these comments again would imrove code quality.

Reply:
I agree that comments should not just retell self-explanatory code. If it does not help explaining it, remove the comment.

Reply:
Okay, the latest commit removes these comments.